### PR TITLE
1.14 doc fixes

### PR DESCRIPTION
**Pull request by Nadine2016**
* Update _index.md

* bump doc hugo theme for GA4 release

* clarify extraction template function

* bump doc hugo theme for GTM

* Update _index.md

* Update _index.md

* Update _index.md

* Update istio.md

* Adding note about gateway/route buffer sizes

* Update _index.md

* Create doc-fixes.yaml

* Rename doc-fixes.yaml to doc-fixes.yaml

* Update docs/content/guides/security/tls/mtls/_index.md

* Trigger CI

---------

# Description

Please include a high level summary of the changes.

This bug fixes ... \ This new feature can be used to ...

_Fill out any of the following sections that are relevant and remove the others_
## API changes
- Added x field to y resource
- ...

## Code changes
- Fix error in `Foo()` function
- Add `Bar()` function
- ...

## CI changes
- Adjusted schedule for x job
- ...

## Docs changes
- Added guide about feature x to public docs
- Updated README to account for y behavior
- ...

# Context

Users ran into this bug doing ... \ Users needed this feature to ...

See slack conversation [here](https://solo-io-corp.slack.com/archives/some/post)

## Interesting decisions
 
We chose to do things this way because ...

## Testing steps

I manually verified behavior by ...

## Notes for reviewers

Be sure to verify intended behavior by ...

Please proofread comments on ...

This is a complex PR and may require a huddle to discuss ...

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by artberger — DISMISSED** [review]

File: docs/content/guides/security/tls/mtls/_index.md [hunk 1]
```diff
@@ -241,10 +241,18 @@ The configuration for the extauth envoy sidecar can be found in the rate-limit-s
 
 ## Cert Rotation
 
-Cert rotation can be done by updating the gloo-mtls-certs secret. The SDS sidecar will
-automatically pick up the change.
+Cert rotation can be done by updating the `gloo-mtls-certs` secret. The SDS sidecar automatically picks up the change.
 
-If you want to automatically rotate certs based on a schedule, you can enable a cert rotation CronJob. In your Helm values file for Gloo Edge installation, set `gateway.certGenJob.cron.enabled` to `true`, and specify a rotation schedule:
+If you want to automatically rotate certificates based on a schedule, you can use the Gloo Edge `gloo-mtls-certgen-cronjob` CronJob. The job is configured to rotate certificates in stages to minimize the downtime for your apps. You have the option to instruct Gloo Edge to wait between stages to ensure that your workloads have enough time to pick up certificate changes. The job follows the following steps:
```
Comment (jenshu) [inline docs/content/guides/security/tls/mtls/_index.md:246]: we technically don't need these changes here since this is only available in 1.15+

Comment (jenshu) [reply docs/content/guides/security/tls/mtls/_index.md:246]: which reminds me, we forgot to add that 1.15+ notice in the docs

**Review by jenshu — APPROVED** [review]

**Review by artberger — APPROVED** [review]

**Pull request merged**